databricks/sdk/azure.py

```diff
@@ -16,18 +17,23 @@ class AzureEnvironment:
 
 ENVIRONMENTS = dict(
     PUBLIC=AzureEnvironment(name="AzurePublicCloud",
+                            databricks_hostname_suffix='.azuredatabricks.net',
                             service_management_endpoint="https://management.core.windows.net/",
                             resource_manager_endpoint="https://management.azure.com/",
                             active_directory_endpoint="https://login.microsoftonline.com/"),
-    GERMAN=AzureEnvironment(name="AzureGermanCloud",
-                            service_management_endpoint="https://management.core.cloudapi.de/",
-                            resource_manager_endpoint="https://management.microsoftazure.de/",
-                            active_directory_endpoint="https://login.microsoftonline.de/"),
+    GERMAN=AzureEnvironment(
+        name="AzureGermanCloud",
+        databricks_hostname_suffix='.databricks.azure.de', # TODO: i'm not sure about this one
```

Comment:
I think this is OK because we don't run Databricks in this cloud (and it is being deprecated: https://learn.microsoft.com/en-us/previous-versions/azure/germany/germany-welcome)